runelite-client/src/main/java/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
```

Comment:
I would also suggest to delete this file.

Reply:
These manifest files are still not deleted